README.md

```diff
@@ -37,14 +37,16 @@ Terraform can be configured by adding the following to your ~/.terraformrc file.
 
 ```hcl
 provider_installation {
-  dev_overrides {
-      "registry.terraform.io/hashicorp/biganimal" = "/Users/<YOUR_HOME>/.terraform.d/plugins/hashicorp.com/edu/biganimal/0.3.1/<OS_ARCH>"
-  }
-
   # For all other providers, install them directly from their origin provider
   # registries as normal. If you omit this, Terraform will _only_ use
   # the dev_overrides block, and so no other providers will be available.
```

Comment:
```suggestion
  # For all other providers, install them directly from their origin provider
  # registries as normal. Until terraform-provider-biganimal is registered 
  # in the terraform registry, we can use a filesystem_mirror.
```

Reply:
thanks! @serdardalgic , it's something I forgot to change